api/graphql/property.py

```diff
@@ -0,0 +1,56 @@
+import typing
+import strawberry
+from strawberry.dataloader import DataLoader
+from .transaction import Transaction
+from .postcode import Postcode
+from ..orm import Property as PropertyORM, Postcode as PostcodeORM
+
+# Now, you need to define a function to load the postcode data based on keys
+async def batch_load_postcodes(keys: typing.List[str]) -> typing.List[Postcode]:
+    print('keys', keys)
+    # Assuming PostcodeORM has a method to fetch multiple postcodes by a list of keys
+    postcode_records = (
+        PostcodeORM
+            .select()
+            .where(PostcodeORM.postcode.in_(keys))
+            .execute()
+    )
+
+    print('postcode_records', postcode_records);
+    print('dicts', postcode_records.__dict__)
+    # Convert database records to Postcode objects
+    postcode_data = [Postcode(**record.__dict__) for record in postcode_records]
```

Comment:
from my limited understanding of python, if this doesn't work, try record.__data__ i tried it in debug and it seems to give some results